src/explorer.ts

```diff
@@ -101,8 +105,9 @@ class DummyObject implements KubernetesObject {
     }
 }
 
-class KubernetesContext implements KubernetesObject {
-    constructor(readonly id: string, readonly metadata?: kubectlUtils.KubectlContext) {
+class ClusterContext implements KubernetesObject {
```

Comment:
I don't get the value in renaming this.  If we're going to rename it, I'd suggest something like `ContextNode` or `ContextObject` or `KubernetesContextObject`.

Reply:
fixed.